### ISSUE TITLE
Improve steps accessibility

### ISSUE THREAD

**Issue by alexstine** [wceu/contributor-orientation-tool#64]
Currently, the steps are not really as accessible as they could be. These suggestions could bring much better functionality for all.
1. The ":" should be after the step number, not prepended to the word.
Wrong: STEP 1 :Use of WordPress
Right: STEP 1: Use of WordPress
Details like this can make a world of difference in how screen readers speak.
2. The steps should have some type of indicator to screen readers about which one is currently selected/in view. You could try adding `aria-current` to the active step `<li>` tag.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

**Comment by dskdipesh-eng** [comment]
1. ****
